frontend/src/components/productCard/ProductCard.tsx

```diff
@@ -25,6 +23,11 @@ export const ProductCard = ({ item }: ProductCardProps) => {
         return true;
     };
 
+    const checkIsBidHistory = () => {
+        if (item.bidHistory.length === 0) return false;
```

Comment:
could also be written like this, i think: `const hasBidHistory = item.bidHistory.length` since `0` is falsy